doc/getting_started/installation.md

```diff
@@ -45,6 +45,10 @@ pip install panel
 
 ::::
 
+:::{important}
+Is Panel installed together with JupyterLab/Jupyter Notebook in your working environment? If not, you need to make sure that `pyviz_comms` is explicitly installed next to JupyterLab/Jupyter Notebook (`conda install pyviz_comms` or `pip install pyviz-comms`) for bi-directional communication to be fully working.
```

Comment:
... "in the same python environment as jupyterlab/ jupyter notebook" is much clearer to me. See also comment below.